specification/datashare/resource-manager/Microsoft.DataShare/stable/2020-09-01/DataShare.json

```diff
@@ -0,0 +1,6755 @@
+        "dataSetId": {
+          "description": "Unique id for identifying a data set resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "kustoDatabaseResourceId": {
+          "description": "Resource id of the kusto database.",
+          "type": "string"
+        },
+        "location": {
+          "description": "Location of the kusto cluster.",
+          "type": "string",
+          "readOnly": true
+        },
+        "provisioningState": {
+          "description": "Provisioning state of the kusto database data set.",
+          "enum": [
+            "Succeeded",
+            "Creating",
+            "Deleting",
+            "Moving",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "OperationList": {
+      "description": "List response for get operations.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The Url of next result page.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Collection of items of type DataTransferObjects.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OperationModel"
+          }
+        }
+      }
+    },
+    "OperationMetaLogSpecification": {
+      "description": "log specifications for operation api",
+      "type": "object",
+      "properties": {
+        "blobDuration": {
+          "description": "blob duration of the log",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "localized name of the log category",
+          "type": "string"
+        },
+        "name": {
+          "description": "name of the log category",
+          "type": "string"
+        }
+      }
+    },
+    "OperationMetaMetricSpecification": {
+      "description": "metric specifications for the operation",
+      "type": "object",
+      "properties": {
+        "aggregationType": {
+          "description": "aggregation type of metric",
+          "type": "string"
+        },
+        "dimensions": {
+          "description": "properties for dimension",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DimensionProperties"
+          }
+        },
+        "displayDescription": {
+          "description": "description of the metric",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "localized name of the metric",
+          "type": "string"
+        },
+        "enableRegionalMdmAccount": {
+          "description": "enable regional mdm account",
+          "type": "string"
+        },
+        "fillGapWithZero": {
```

Comment:
What is this value for? I don't see an example using this value, is this meant to be a boolean?